src/components/login.jsx

```diff
@@ -1,8 +1,128 @@
+import React, { useState } from 'react';
+import { MDBContainer, MDBTabs, MDBTabsItem, MDBTabsLink, MDBTabsContent, MDBTabsPane, MDBBtn, MDBIcon, MDBInput, MDBCheckbox } from 'mdb-react-ui-kit';
+
+import 'mdb-react-ui-kit/dist/css/mdb.min.css';
+
 export const Login = ({component}) => {
-    return <>
-        {
-            component=="login"? <h1>login</h1> : <h1>logout</h1>
+    const [justifyActive, setJustifyActive] = useState('tab1');;
+    const handleJustifyClick = (value) => {
```

Comment:
this method will be more readable if you change it to 
if (value !== justifyActive) {
setJustifyActive(value);
}